pagopa/selfcare-onboarding
# [SELC-3159] feat: send mail for approve on onboarding and registration

#### List of Changes



* Added a new asynchronous step to the workflow.
* Integrated functionality to trigger email delivery for approving registration and onboarding by admin

#### Motivation and Context



It enables an email dispatch during the onboarding of an organization from the frontend when it need of approving by admin.

#### How Has This Been Tested?





1. Navigate to the organization onboarding section in the frontend.
2. Complete the onboarding process for a test organization that need of approving by admin.
3. Verify that the email notification is sent promptly.

#### Screenshots (if appropriate):

#### Types of changes




#### Checklist:
